projects/hslayers/src/components/draw/draw.service.ts

```diff
@@ -159,6 +159,12 @@ export class HsDrawService {
       this.isAuthorized =
         endpoint.user !== 'anonymous' && endpoint.user !== 'browser';
     });
+
+    this.HsEventBusService.LayerManagerLayerVisibilityChanges.subscribe((event) => {
+      if (this.draw && event.layer.ol_uid == this.selectedLayer.ol_uid){
```

Comment:
Why cant we not use the actual layers instead of ol_uid?

Reply:
Just wanted to keep it simple , not compare objects